run_uts.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -x
+set -e
+date
```

Comment:
This is a convention I've been carrying over from a jenkins config. I quite like them to be honest; they're cheap, and can save some pain in the case where you need that info.